.idea/workspace.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this should be in your global gitignore configuration ;-)

can you please remove it from the pull request?

Reply:
cool. I'll add this to my global gitignore configuration.
and .idea remoed already